packages/playback-core/src/index.ts

```diff
@@ -582,13 +613,14 @@ export const getStreamTypeConfig = (streamType?: ValueOf<StreamTypes>) => {
 };
 
 export const getDRMConfig = (
-  props: Partial<Pick<MuxMediaPropsInternal, 'src' | 'playbackId' | 'drmToken' | 'customDomain'>>
+  props: Partial<Pick<MuxMediaPropsInternal, 'src' | 'playbackId' | 'drmToken' | 'customDomain' | 'drmTypeCb'>>
 ): Partial<HlsConfig> => {
   const {
     drmToken,
-    src,
-    playbackId = toPlaybackIdFromSrc(src), // Since Mux Player typically sets `src` instead of `playbackId`, fall back to it here (CJP)
+    playbackId: playbackIdWithOptionalParams, // Since Mux Player typically sets `src` instead of `playbackId`, fall back to it here (CJP)
+    drmTypeCb,
   } = props;
+  const [playbackId] = playbackIdWithOptionalParams ? toPlaybackIdParts(playbackIdWithOptionalParams) : [];
```

Comment:
NOTE: Need to account for the potential additional params in playback-core because of our `<mux-video>` and `<mux-audio>` implementation of `playbackId`...